lib/inspec/config.rb

```diff
@@ -81,8 +83,9 @@ def unpack_train_credentials
       credentials.merge!(_utc_generic_credentials)
 
       _utc_determine_backend(credentials)
-      credentials.merge!(Train.unpack_target_from_uri(final_options[:target] || '')) # TODO: this will be replaced with the credset work
       transport_name = credentials[:backend].to_s
+
+      _utc_merge_credset(credentials, transport_name)
```

Comment:
@clintoncwolfe  just want to understand why we are using _ in front of methods. Is it because we want to define it private ?

Reply:
For me, it means that the method is the result of a linter-forced refactoring.  In other words, to build the functionality of unpack_train_credentials, I needed to break it into many small pieces that would have no possible use to anything else.

If it turns out that one of the _utc methods are useful outside of the context of unpacking train credentials, I'd rename it.